Jenkinsfile

```diff
@@ -4,16 +4,23 @@ pipeline {
     }
 
     environment {
-        ET_EUS_API = "http://selenoid:4444/wd/hub"
-        SUT_URL = "$WORKSPACE/sut"
+        SELENOID_URL = "http://selenoid:4444/wd/hub"
+        SUT_LOCATION = "$WORKSPACE/sut"
         E2ESUITE_URL = "$WORKSPACE"
+        IP_SERVER="156.35.119.57"
```

Comment:
Do not place hardcoded addresses like this in the jenkinsfile. Use docker network addresing to locate the services.